scripts/windowsScript.ps1

```diff
@@ -0,0 +1,23 @@
+#
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+
+git submodule update --init -- jni/external/nmslib
+git submodule update --init -- jni/external/faiss
+
+git apply patches/windows/CMakeLists.patch --verbose
+
+# Validating if the CMakeLists patch is applied
+echo "Validating if the CMakeLists patch is applied"
+type jni/CMakeLists.txt | Select-String "Windows"
+
+# Replace '_MSC_VER' with '__MINGW32__'
```

Comment:
These comments should explain why we are replacing them instead of what the commands are doing.

Reply:
updated